config/settings.yml

```diff
@@ -242,6 +242,7 @@ modules_appeals_api:
     region: "region"
     bucket: "bucket"
   status_simulation_enabled: false
+  legacy_appeals: false
```

Comment:
You could set this to true so that you don't have to worry about forcing it to true in the `with_settings` of your specs. Remember there's another repo that controls the value here per env, the vets-api repo's config/settings.yml is the example for local development.

Reply:
the ticket says set to false